CONTRIBUTING.md

```diff
@@ -6,8 +6,8 @@ By contributing to this repository you agree to license your contribution under
 
 When submitting a pull request (PR), please use the following guidelines:
 
-- First verify your changes by running `bundle exec jekyll serve -w`.
-- For most webpage changes, please submit pull requests to this repository based off the "src" branch.
```

Comment:
I saw PRs merged to `master` and couldn't find a `src` branch, so assuming that this is outdated.